<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Dag Andersen &lt;danders@get2net.dk&gt;</author>
 <class>KPlatoWork::TaskCompletionPanel</class>
 <widget class="QWidget" name="KPlatoWork::TaskCompletionPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>272</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="started">
        <property name="text">
         <string>Started:</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="startTime">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="finished">
        <property name="text">
         <string>Finished:</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="finishTime">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Edit Mode:</string>
        </property>
        <property name="buddy">
         <cstring>editmode</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="editmode">
        <item>
         <property name="text">
          <string>Calculate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manual entry</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addEntryBtn">
       <property name="text">
        <string>Add Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeEntryBtn">
       <property name="text">
        <string>Remove Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KPlato::CompletionEntryEditor" name="entryTable">
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPlato::CompletionEntryEditor</class>
   <extends>QTableView</extends>
   <header>kptusedefforteditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>started</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>finished</tabstop>
  <tabstop>finishTime</tabstop>
  <tabstop>editmode</tabstop>
  <tabstop>addEntryBtn</tabstop>
  <tabstop>removeEntryBtn</tabstop>
  <tabstop>entryTable</tabstop>
 </tabstops>
 <includes>
  <include location="local">kdatetimewidget.h</include>
  <include location="local">kdatewidget.h</include>
  <include location="local">knuminput.h</include>
  <include location="local">kptusedefforteditor.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
